src/main/java/net/snowflake/ingest/SimpleIngestManager.java

```diff
@@ -199,20 +244,31 @@ private void init(String account, String user, String pipe, KeyPair keyPair) {
    * <p>This method is deprecated, please use the constructor that only requires PrivateKey instead
    * of KeyPair.
    *
-   * @param account The account into which we're loading Note: account should not include region or
-   *     cloud provider info. e.g. if host is testaccount.us-east-1.azure .snowflakecomputing.com,
-   *     account should be testaccount
+   * @param account the account into which we're loading Note: account should not include region or
+   *     cloud provider info. e.g. if host is testaccount.us-east-1.azure .snowflakecomputing.com
+   *     account should be testaccount If this is the case, you should use the constructor that
+   *     accepts hostname as argument
    * @param user the user performing this load
    * @param pipe the fully qualified name of the pipe
    * @param keyPair the KeyPair we'll use to sign JWT tokens
+   * @param schemeName http or https
+   * @param hostName the hostname
+   * @param port the port number
    */
   @Deprecated
-  public SimpleIngestManager(String account, String user, String pipe, KeyPair keyPair) {
```

Comment:
Will this be an API change if we remove this constructor?

Reply:
Sorry for confusion, I rearranged some ctor's. No Behavior change.